.changeset/hip-flies-dance.md

```diff
@@ -0,0 +1,5 @@
+---
+"@preconstruct/cli": patch
+---
+
+Pass config file path to typescript so relative paths are applied to correct directory
```

Comment:
```suggestion
The `outDir` and `declarationDir` tsconfig options are now ignored. These options are unnecessary for Preconstruct since it controls where the `.d.ts` files are emitted to. This fixes confusing errors if you had these options set.
```